scripts/deploy.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+import os
+import sys
+from time import time, sleep
+import random
+sys.path.append(os.path.dirname(__file__))  # noqa
+
+import hypothesistooling as tools
+
+
+if __name__ == '__main__':
+    last_release = tools.latest_version()
+
+    print("Current version: %s. Latest released version: %s" % (
+        tools.__version__, last_release
+    ))
+
+    start_time = time()
+
+    prev_pending = None
+
+    while time() <= start_time + 60 * 60:
+        jobs = tools.build_jobs()
+
+        failed_jobs = [
+            v for k, vs in jobs.items() if k not in ('started', 'passed')
```

Comment:
The intent of this section seems clear enough (from variable name), but I didn't find it scanning too well

Is there a way of framing/lining it better, or not really?